### PR TITLE
OIDC Auth 2.1.1

### PR DESCRIPTION
**Pull request by tjuerge**
Updated OpenID Connect Authentication Plugin to v2.1.1:

* fixes some issues with the newly added auto-login servlet filter

### CODE REVIEW

File: authoidc.properties [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
-archivedVersions=1.0.4,1.1.0,2.0.0
-publicVersions=2.1.0
+archivedVersions=1.0.4,1.1.0,2.0.0,2.1.0
+publicVersions=2.1.1
 
 defaults.mavenGroupId=org.vaulttec.sonarqube.auth.oidc
 defaults.mavenArtifactId=sonar-auth-oidc-plugin
 
+2.1.1.description=Fixes some issues with the newly added auto-login servlet filter
+2.1.1.sqVersions=[7.9,LATEST]
```
Comment (ganncamp) [inline authoidc.properties:11]: This range needs to start with 8.9. 
7.9 is fully past EOL and sunsetted now.

File: authoidc.properties [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
+2.1.1.date=2021-10-24
```
Comment (ganncamp) [inline authoidc.properties:12]: This date doesn't look right to me. You posted the request in the community on 28 December.

**Review by ganncamp — CHANGES_REQUESTED** [review]

File: authoidc.properties (hunk 1 above)
Comment (tjuerge) [reply authoidc.properties:11]: This plugin is [compiled and tested against SQ 7.4](https://github.com/vaulttec/sonar-auth-oidc/blob/master/pom.xml#L26). But as requested I'll limit this plugin's marketplace entry to SQ 8.9 and newer.

File: authoidc.properties (hunk 2 above)
Comment (tjuerge) [reply authoidc.properties:12]: Yes, it took me 2 month (after the release of this plugin version on 2021-10-24) to prepare the steps for the marketplace entry :)
Screenshot 2022-01-06 at 23 15 19

Comment (ganncamp) [reply authoidc.properties:12]: This date should be (approximately) when the plugin became available in the Marketplace.

**Review by ganncamp — CHANGES_REQUESTED** [review]

File: authoidc.properties (hunk 2 above)
Comment (tjuerge) [reply authoidc.properties:12]: Ok, I'll fix this...

**Review by ganncamp — APPROVED** [review]

**Pull request merged**